buildlib/tools/builds.sh

```diff
@@ -261,7 +261,13 @@ build_clang() {
 # Build with gcc-latest module
 #
 build_gcc() {
-	cflags=$1
+	while true; do
```

Comment:
can pass configure args as $@ to build_gcc, like so:
```
build_gcc {
    ...
    ${WORKSPACE}/contrib/configure-devel $@
    ...
}
...

build_gcc CFLAGS=-Og CXXFLAGS=-Og
build_gcc --with-devx=no
```

Reply:
Done